src/Compilers/Core/Portable/Emit/InstrumentationKind.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.Emit
+{
+    public enum InstrumentationKind
+    {
+        None = 0,
+        TestCoverage = 1,
```

Comment:
I don't have a strong opinion about whether instrumentation kinds are encoded using an enum or as strings--whatever gets through review is fine with me.

An advantage of an enum is that the allowed values are explicit, and that's my preference, but mostly we need something that works merged as soon as practical.